### ISSUE TITLE
Navigating to a previous page again doesn't seem to work

### ISSUE THREAD

**Issue by juristr** [ionic-team/ionic#15295]
# Bug Report

**Ionic Info**
Run `ionic info` from a terminal/cmd prompt and paste the output below.

```
Ionic:

   ionic (Ionic CLI)          : 4.1.1 (/Users/juri/node_modules/ionic)
   Ionic Framework            : @ionic/angular 4.0.0-beta.3
   @angular-devkit/core       : 0.7.4
   @angular-devkit/schematics : 0.7.4
   @angular/cli               : 6.1.4
   @ionic/ng-toolkit          : 1.0.7
   @ionic/schematics-angular  : 1.0.5

Cordova:

   cordova (Cordova CLI) : not installed
   Cordova Platforms     : not available
   Cordova Plugins       : not available

System:

   ios-sim : 5.0.2
   NodeJS  : v9.11.1 (/Users/juri/.nvm/versions/node/v9.11.1/bin/node)
   npm     : 5.6.0
   OS      : OS X El Capitan
   Xcode   : Xcode 7.3.1 Build version 7D1014
```

**Describe the Bug**
When I navigate from a list to the detail, then click the back button navigate again to the detail view it doesn't open. No error on the console either.

**Steps to Reproduce**
1. Clone the repo: https://github.com/juristr/ionic4-experiments
1. Launch the app, click the "Plants" link on the `/home` page
1. Click on the first list entry
1. click the back button on the detail page
1. Click again on a list entry -> doesn't work.

Similar things happen when clicking on the "edit" button in the Detail page and then clicking "Back" and then again trying to enter the Edit view.

**Related Code**
https://github.com/juristr/ionic4-experiments

**Expected Behavior**
I expect to be able to enter the previous view again, or a stacktrace/error on the console if my configuration or navigation approach is wrong 😉 

**Additional Context**

@mhartington as discussed previously on slack 👍thx

**Comment by wesleyxiao** [comment]
same issue happen, and console log show : router-outlet destroyed.

**Comment by mhartington** [comment]
Go home Ionitron...you're drunk...

**Comment by liamdebeasi** [comment]
Hi there,

This has been fixed with the release of Ionic 4.2.0. Please upgrade to that version, and let me know if you are still running into issues.

Thanks for using Ionic!
